message SearchResultMessage 
{
	required string picture_name = 1;
	required uint32 result_length = 2;
    required uint32 task_id = 3;

}
